### DISCUSSION TITLE
Is there a GPS-location of the vehicle available?

### DISCUSSION THREAD

**Discussion by binderth** [tillsteinbach/WeConnect-mqtt#312] [Q&A]
Is there a GPS location available from the API and is there a MQTT-topic for that? I didn't find it in my use case (ID.3 MY 2020)?

**Comment by tillsteinbach** [comment]
Only a position when parking

**Reply by binderth** [reply]
That's OK, I couldn't find a topic for that, can you please point me to it? ;)

**Comment by Domoberry** [comment]
The following topics might be what your looking for:

`weconnect/0/vehicles/WVWxxxxxxxxxxxxxx/parking/parkingPosition/latitude`
`weconnect/0/vehicles/WVWxxxxxxxxxxxxxx/parking/parkingPosition/longitude`

and possibly also:
`weconnect/0/vehicles/WVWxxxxxxxxxxxxxx/parking/parkingPosition/carCapturedTimestamp`

@others: please correct if these are not the right topics, above is just from my experience on ID3 MY '20 with software 3.2.

**Reply by binderth** [reply]
> @Others: please correct if these are not the right topics, above is just from my experience on ID3 MY '20 with software 3.2.

ah! Mine got version 3.0. this could explain the missing lat/long... the 'carCapturedTimestamp' is present with my environment.
How can I upgrade the software? Do I have to go to the dealership?

**Reply by Domoberry** [reply]
That was a larger update done by the dealer some time ago, it was not suitable for an online update I recall.

**Comment by udo1toni** [comment]
And please be aware that you have to
a) allow sharing that data (i.e. set the corresponding switch in the ID account and set the switch in the car itself if I remember it correctly...)
b) set a pin and set this pin also in the docker-compose.yaml
